config/settings.py

```diff
@@ -477,6 +477,10 @@ def show_toolbar(request):
 # GOOGLE TAG ADWORDS
 GOOGLE_ADWORDS_ACTIVATE = env.bool("GOOGLE_ADWORDS_ACTIVATE", default=False)
 
+# CRISP
+CRISP_WEBSITE_ID = env.str("CRISP_WEBSITE_ID")
+CRISP_ACTIVATED = env.bool("CRISP_ACTIVATED")
```

Comment:
pour débloquer les tests 

```suggestion
CRISP_ACTIVATED = env.bool("CRISP_ACTIVATED", default=False)
```
<img width="1060" alt="image" src="https://github.com/MTES-MCT/sparte/assets/16051000/404918c1-5093-4e9a-9a62-f61b6219a1e8">